### PR TITLE
add bikeRatings query and comments property to Rating.js

### PR DESCRIPTION
**Pull request by tjprange**

### CODE REVIEW

File: src/api/bikeRatings.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  const db = firebase.firestore();
+  const ratings = [];
+
+  const snapshot = await db.collection('ratings').where('bike_id', '==', bikeId).get();
+  snapshot.forEach((doc) => {
+    ratings.push(parseFloat(doc.data().rating));
+  });
+
+  const total = ratings.reduce((accumulator, currentValue) => accumulator + currentValue);
+  return total / ratings.length;
```
Comment (anderja8) [inline src/api/bikeRatings.js:16]: Not how I would have thought to do this, but super nifty!

File: src/models/Rating.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
 // class definition for Bike entity
 export default class Rating {
-  constructor(id, bikeId, rating) {
+  constructor(id, bikeId, rating, comment) {
```
Comment (anderja8) [inline src/models/Rating.js:3]: Awesome!

File: src/screens/HomeStack/BrowseBikes.jsx [hunk 3]
```diff
@@ -2,17 +2,12 @@ import React, { useState, useEffect } from 'react';
...
-  Row,
-  Text,
-  Spinner
-} from 'native-base';
+import { Container, Content, Grid, Row, Text, Spinner } from 'native-base';
 import BrowseBikesMap from '../../components/BrowseBikesMap';
 import BrowseBikesList from '../../components/BrowseBikesList';
 import { getBikesWithinRadius } from '../../api/bikes';
+// eslint-disable-next-line no-unused-vars
+import { bikeRatings } from '../../api/bikeRatings';
```
Comment (anderja8) [inline src/screens/HomeStack/BrowseBikes.jsx:10]: I don't think we need to import something if it's not being used. I'm going to merge this in and fix it in my PR though so we don't lose velocity over something that's just a small nitpick.

**Review by anderja8 — APPROVED** [review]
I left one nitpick about importing bikeRatings in BrowseBikes.jsx that I will edit in my PR that I'll post in a few minutes. Overall though, awesome job!

File: src/api/bikeRatings.js (hunk 1 above)
Comment (tjprange) [reply src/api/bikeRatings.js:16]: I saw this earlier in the day so I wanted to give it some use before I forgot all about it.  Helps for remembering!

**Pull request merged**